Transport/GpsTransportFactory.php

```diff
@@ -28,8 +31,13 @@ public function createTransport(string $dsn, array $options, SerializerInterface
     {
         $options = $this->gpsConfigurationResolver->resolve($dsn, $options);
 
+        $clientConfig = $options->getClientConfig();
+        if (null !== $this->cache && ! isset($clientConfig['authCache'])) {
```

Comment:
We can do this with coalescing assign operator supported since PHP7.4.
```php
if ($this->cache instanceof CacheItemPoolInterface) {
   $clientConfig['authCache'] ??= $this->cache;
}
```

Reply:
Nice suggestion, thanks.